cpp/splinepy/py/py_spline_reader.hpp

```diff
@@ -40,16 +41,12 @@ class BSplineParser {
 
   using SplineEntry = input_output::SplineEntry;
 
-  /// Counter variable
-  int i;
-  /// Counter variable for inner loops
-  int j;
-
   BSplineParser() {}
 
   /// @brief Convert BSpline to Python dictionary
   /// @param bspline BSpline object
-  py::dict bspline_to_dict(SplineEntry const& bspline) {
+  py::dict BsplineToDict(SplineEntry const& bspline) {
```

Comment:
```suggestion
  py::dict BSplineToDict(SplineEntry const& bspline) {
```
same for the other ones. (Bspline -> BSpline)